unified_planning/engines/factory.py

```diff
@@ -40,6 +40,7 @@
     "enhsp": ("up_enhsp.enhsp_planner", "ENHSPSatEngine"),
     "enhsp-opt": ("up_enhsp.enhsp_planner", "ENHSPOptEngine"),
     "tamer": ("up_tamer.engine", "EngineImpl"),
+    "aries": ("up_aries", "Aries"),
```

Comment:
Is `up_aries` available in pypi?

And when you add an engine in the `DEFAULT_ENGINES` dict, you also need to add it in the `DEFAULT_ENGINES_PREFERENCE_LIST`.

Reply:
`up_aries` is currently not available in pypi yet.

I have updated the requested changes.

Reply:
We have the policy to add an engine as a default engine only when it is available in pypi

Reply:
@alvalentini I have moved the grpc planner definition to PR #334. Once the PR is merged, I will setup PyPI for `up_aries`